GoogleUtilities/Environment/GULAppEnvironmentUtil.m

```diff
@@ -15,30 +15,16 @@
 #import "GoogleUtilities/Environment/Public/GoogleUtilities/GULAppEnvironmentUtil.h"
 
 #import <Foundation/Foundation.h>
-#import <dlfcn.h>
-#import <mach-o/dyld.h>
+#import <objc/runtime.h>
 #import <sys/sysctl.h>
 #import <sys/utsname.h>
-#import <objc/runtime.h>
+
+#import "third_party/Environment/Public/IsAppEncrypted.h"
```

Comment:
Why does this header need to be public?

Reply:
Because I moved it to it's own SPM target, the target then needed a public headers directory. And I created the target because I moved the 3P code to the root of repo.

In the first commit (e274d3b), I tried a more localized approach where I kept the 3P code within the GoogleUtilities-Environment sources directory. This was also valid, but I felt that the 3P code was still buried deep within the repo– and I personally like the 3p directory at the root.

Reply:
SGTM. Also, GoogleUtilities is documented for Google use only.

Reply:
> Also, GoogleUtilities is documented for Google use only.

Yep, the target I've created doesn't have a `.product` specification so it shouldn't really be an artifact that clients outside of the GULs package can use.